config/puma.rb

```diff
@@ -48,24 +46,28 @@
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
 # and maximum; this matches the default thread size of Active Record.
 #
-max_threads = ENV.fetch('RAILS_MAX_THREADS', 5).to_i
-threads ENV.fetch('RAILS_MIN_THREADS', max_threads).to_i, max_threads
+max_threads_count = ENV.fetch('RAILS_MAX_THREADS') { 5 }
```

Comment:
no need to case this as an integer anymore? looks like it's still happening on line 56

Reply:
The default Rails 6.1 puma config is:

```
max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
threads min_threads_count, max_threads_count
worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
```

so I figured threads can handle strings but idk about worker_timeout